### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by JordanMartinez**
Backlinking to purescript-contrib/governance#35

### CODE REVIEW

File: src/Data/ArrayBuffer/Types.purs [hunk 1]
```diff
@@ -15,7 +15,7 @@ type ByteOffset = Int
 -- | Length in bytes of a DataView or ArrayBuffer
 type ByteLength = Int
 
-foreign import kind ArrayViewType
+foreign import data ArrayViewType
```
Comment (thomashoneyman) [inline src/Data/ArrayBuffer/Types.purs]: Should this just be `data ArrayViewType`? My memory is a bit hazy on this.

Comment (JordanMartinez) [reply src/Data/ArrayBuffer/Types.purs]: Ah... Yeah, it should. Sorry about that.

Comment (thomashoneyman) [reply src/Data/ArrayBuffer/Types.purs]: Admittedly I don't have a firm grasp of polykinds at the moment, and I can't find a source to back me up, but I thought `foreign import kind X` became just `data X`. Perhaps @natefaubion knows the full explanation.

**Comment by thomashoneyman** [comment]
```
Run spago install
spago: 
↳ ./spago.dhall
  ↳ ./packages.dhall
    ↳ https://raw.githubusercontent.com/purescript/package-sets/prepare-0.14/src/packages.dhall sha256:04c542b4d44c644031baf6220f441718289302ff4d4f71d43d57887eb295cf26

Error: Import integrity check failed

Expected hash:

↳ 04c542b4d44c644031baf6220f441718289302ff4d4f71d43d57887eb295cf26

Actual hash:

↳ f9b49a477fcd822a7119413b689123f8e1694b0008c156971f1931751079a56f


2│       https://raw.githubusercontent.com/purescript/package-sets/prepare-0.14/src/packages.dhall sha256:04c542b4d44c644031baf6220f441718289302ff4d4f71d43d57887eb295cf26
```

**Comment by JordanMartinez** [comment]
Yeah. This is because I updated `numbers` to include `functions` as a dependency. However, the SHA hash was from the package set before that change was made.

File: src/Data/ArrayBuffer/Types.purs (hunk 1 above)
Comment (JordanMartinez) [reply src/Data/ArrayBuffer/Types.purs]: Actually, I don't remember myself...

**Comment by thomashoneyman** [comment]
This looks good other than reverting the whitespace changes to the ci.yml file!

File: src/Data/ArrayBuffer/Types.purs (hunk 1 above)
Comment (JordanMartinez) [reply src/Data/ArrayBuffer/Types.purs]: Based on [what I wrote previously](https://github.com/JordanMartinez/purescript-jordans-reference/blob/latestRelease/11-Syntax/03-Type-Level-Programming-Syntax/ps-0.14/02-Basic-Syntax/01-Defining-Custom-Kinds.purs#L9-L11), I think this is correct?

Comment (thomashoneyman) [reply src/Data/ArrayBuffer/Types.purs]: Yep! I think once we revert the whitespace changes to ci.yml we can merge it.

**Comment by JordanMartinez** [comment]
I decided it was easier to rerun my script on a fresh copy and update it to get rid of some of those unneeded commits.

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**